python/tvm_ffi/_ffi_api.py

```diff
@@ -16,6 +16,37 @@
 # under the License.
 """FFI API."""
 
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
 from . import registry
 
+if TYPE_CHECKING:
```

Comment:
Alternatively, we may:
- Option A) Suppress type errors on all `_ffi_api.*` access
- Option B) Auto generate stubs in another `*.pyi` file